mkosi/bootloader.py

```diff
@@ -72,8 +72,11 @@ def want_grub_efi(context: Context) -> bool:
     if context.config.bootloader != Bootloader.grub:
         return False
 
+    if not (arch := context.config.architecture.to_grub()):
+        return False
```

Comment:
Maybe print a warning here?

Reply:
No this gets invoked more than once, it'd lead to noise